frontend/public/components/pod-exec.jsx

```diff
@@ -194,16 +195,21 @@ const PodExec_ = connectToFlags(FLAGS.OPENSHIFT)(
 
       return (
         <div>
+          {infoMessage && infoMessage}
           <div className="co-toolbar">
             <div className="co-toolbar__group co-toolbar__group--left">
               <div className="co-toolbar__item">{t('public~Connecting to')}</div>
               <div className="co-toolbar__item">
-                <Dropdown
-                  className="co-toolbar__item-dropdown"
-                  items={_.mapValues(containers, nameWithIcon)}
-                  title={nameWithIcon(activeContainer || <LoadingInline />)}
-                  onChange={this.onChangeContainer}
-                />
+                {containers.length > 1 ? (
```

Comment:
If we're going to let you switch containers, we should start a pod with all of the original containers and just connect to the right container without creating a whole new pod.

Reply:
Switching containers is not working so I am taking the 3.11 approach and only having 1 container per debug pod.

Reply:
@spadgett - I have switching working now and it is fast and smooth.
![container-switching-debug-pod](https://user-images.githubusercontent.com/18728857/146467573-3d1f7c64-125f-4bad-ae5e-95107ce26704.gif)

Reply:
Switching works now @spadgett